spring-boot-project/spring-boot-actuator-autoconfigure/pom.xml

```diff
@@ -331,6 +331,16 @@
 			<artifactId>spring-session-core</artifactId>
 			<optional>true</optional>
 		</dependency>
+		<dependency>
+			<groupId>org.mongodb</groupId>
```

Comment:
This isn't sorted properly still (m is before s)

Reply:
done